canopeum_backend/canopeum_backend/models.py

```diff
@@ -34,8 +40,11 @@ class User(AbstractUser):
         unique=True,
     )
     USERNAME_FIELD = "email"
-    REQUIRED_FIELDS: ClassVar[list[str]] = []  # type: ignore
-    role = models.ForeignKey(Role, models.RESTRICT, null=False, default=1)  # type: ignore
+    REQUIRED_FIELDS: ClassVar[list[str]] = []
+    role = models.ForeignKey[Role, Role](Role, models.RESTRICT, null=False, default=1)
+    # Missing "id" in "Model" or some base "User" class?
+    if TYPE_CHECKING:
+        id: int
```

Comment:
Not sure if missing from upstream type stubs or if we're doing something specific to have an id on our user.